sotodlib/io/g3tsmurf_utils.py

```diff
@@ -202,3 +204,41 @@ def get_batch(
     except GeneratorExit:
         pass
     
+def load_hwp_data(aman, configs=None, data_dir=None):
+    """Load HWP data that will match with aman timestamps and 
+    interpolate angles into hwp_angle
+    
+    Arguments
+    ----------
+    aman : AxisManager
+        AxisManager of loaded SMuRF data
+    configs : (optional) config filename or dictionary containing "hwp_prefix"
+    data_dir : (optional) the "hwp_prefix" data directory. overrides configs
```

Comment:
Can you point to an example `hwp_prefix` file or dictionary in the doc-string or readthedocs?

Reply:
Actually, I guess you also haven't seen these changes? https://sotodlib.readthedocs.io/en/latest/hwp.html